.changeset/good-trains-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": patch
```

Comment:
should be minor here

Reply:
as it's a fix, it seems like a patch to me, but maybe we have different usage for the apps ?

Reply:
it changes a bit the UI because we remove some extra unecessary permission prompts 🤷‍♂️ so to me `minor` can be justified